src/packs/heavy/Railgun.yml

```diff
@@ -8,9 +8,6 @@ data:
     suppress: true
     type: special
     value: 20
-  bundle:
-    amount: 1
-    bundled: false
```

Comment:
Don't remove data unrelated to PR

Reply:
You helpfully did this in one of your commits.